packages/rocketchat-lib/server/lib/notifyUsersOnMessage.js

```diff
@@ -104,9 +104,24 @@ RocketChat.callbacks.add('afterSaveMessage', function(message, room) {
 	// Update all the room activity tracker fields
 	RocketChat.models.Rooms.incMsgCountAndSetLastMessageById(message.rid, 1, message.ts, RocketChat.settings.get('Store_Last_Message') && message);
 
-	// Update all other subscriptions to alert their owners but witout incrementing
+	// Update all other subscriptions to alert their owners but without incrementing
 	// the unread counter, as it is only for mentions and direct messages
 	RocketChat.models.Subscriptions.setAlertForRoomIdExcludingUserId(message.rid, message.u._id);
+	const subscriptions = RocketChat.models.Subscriptions.findByRoomIdForAlert(message.rid);
+	if (subscriptions) {
```

Comment:
`subscriptions` will be the cursor, so it will always exists

Reply:
Fixed!